### PR TITLE
add release status command

### PR DESCRIPTION
**Pull request by evansde77**
Lays out basic release status command to show status of releases and merges and tagging underlying it. 

Addresses https://github.com/evansde77/cirrus/issues/169

### CODE REVIEW

File: src/cirrus/release_status.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+        # handles caser where tag or release is not present, eg typo
+        if (not tag_commit) and (not branch_commit):
+            msg = (
+                "Unable to find any branch or tag commits "
+                "for release \'{}\'\n"
+                "Are you sure this is a valid release?"
+            ).format(release)
+            LOGGER.error(msg)
+            return
```
Comment (sfoulkes) [inline src/cirrus/release_status.py]: None is falsey but I think you should return result/False here.

File: tests/unit/cirrus/release_status_tests.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        self.mock_conf.gitflow_origin_name = mock.Mock(return_value='origin')
+        self.mock_ghc.config = self.mock_conf
+
+        self.mock_ghc.find_release_commit = mock.Mock(side_effect=["BRANCH_COMMIT", "TAG_COMMIT"])
+        self.mock_ghc.commit_on_branches = mock.Mock(return_value=['master', 'remotes/origin/master'])
+        self.mock_ghc.merge_base = mock.Mock(return_value="MERGE_COMMIT")
+        self.mock_ghc.git_show_commit = mock.Mock(return_value=" this contains release/0.2.3 blah")
+
+    def tearDown(self):
+        self.patch_ghc.start()
```
Comment (sfoulkes) [inline tests/unit/cirrus/release_status_tests.py]: Why are you calling start() here?

Comment (evansde77) [inline tests/unit/cirrus/release_status_tests.py]: copypasta, should be stop()

Comment (evansde77) [inline tests/unit/cirrus/release_status_tests.py]: fixed

File: src/cirrus/release_status.py (hunk 1 above)
Comment (evansde77) [inline src/cirrus/release_status.py]: return False added

**Comment by sfoulkes** [comment]
LGTM

**Pull request merged**